app/assets/javascripts/payments.js

```diff
@@ -0,0 +1,3 @@
+function braintree_setup() {
+  braintree.setup(gon.client_token, "custom", {id: "payment-form"});
```

Comment:
Identifier 'client_token' is not in camel case.<br>'braintree' is not defined.<br>'gon' is not defined.